bin/hdfcoinc/pycbc_page_segtable

```diff
@@ -108,8 +108,11 @@ for segment_name in opts.segment_names:
 
     # put comment in caption
     caption += " " + segment_name
-    if comment_dict[key] != None:
-        caption += " ("+comment_dict[key]+")"
```

Comment:
Look at the code Chris. The variable "key" is set within a for loop above, so the value here is just the exit value, which is not well defined. Specifically in this function we allow for multiple segments (ie ANALYSED&CAT_2). The comment stuff does not know about that. Maybe this was broken some time before?